### ISSUE TITLE
App doesn't get past choosing phase

### ISSUE THREAD

**Issue by rhit-lighthlt** [AdAway/AdAway#2828]
**Describe the bug**
On installation of app it goes to which version you want to run and then once you tap vpn it does nothing.

**To Reproduce**
Open app
Tap vpn

**Expected behavior**
Moves onto next setup screen

**Screenshots**

**Device**
 - AdAway version: 5.x
 - Device: Pixel 5
 - Android version: 11
 - Root version: None

**Additional context**
Add any other context like logcat about the problem here. Use adb logcat if you have developer settings enabled on your device or use any application like [MatLog](https://play.google.com/store/apps/details?id=com.pluscubed.matlog) to save logs. Include the crash report in the issue in a code block, a file attachment, or put it in a gist and provide link to that gist.

**Comment by metzger100** [comment]
Yeah same here. I am using Android 11 Pixel Experience Plus. It doesn't happen anything pressing the VPN-Button on Setup. Somehow it doesn't prompt the VPN-Setupwindow of the OS to add the VPN.

**Comment by a-silvia** [comment]
Same here (fairphone 3, android 11). I had the app installed for several months, opened it and it was unresponsive. I reinstalled it and now similarly get stuck on this screen. Tapping the root version brings up a pop-up, but tapping the VPN gives me no feedback at all.
